python_scripts/linear_models_sol_01.py

```diff
@@ -93,10 +96,10 @@ def goodness_fit_measure(true_values, predictions):
     # we compute the error between the true values and the predictions of our
     # model
     errors = np.ravel(true_values) - np.ravel(predictions)
-    # We have several possible strategy to reduce all errors to a single value.
-    # Computing the mean error (sum divided by the number of element) looks
-    # like a good solution. However, we have negative error and therefore, we
-    # need to sum only positive numbers. Therefore, we can either square each
+    # We have several possible strategies to reduce all errors to a single value.
+    # Computing the mean error (sum divided by the number of element) could seem
+    # like a good solution. However, we have negative errors that will misleadingly
```

Comment:
"could seem like" seems odd to me :)

Reply:
We can alternatively use "[might seem like](https://www.phrasemix.com/phrases/it-might-seem-like-something-but-sentence)", which is a little better. What do you think?

Reply:
+1 for "might seem like" or just "seems like" as you prefer.